llvm/lib/DebugInfo/LogicalView/Core/LVSupport.cpp

```diff
@@ -49,7 +49,7 @@ std::string llvm::logicalview::flattenedFilePath(StringRef Path) {
   std::string Name(Path);
   std::transform(Name.begin(), Name.end(), Name.begin(), tolower);
 
-  const char *CharSet = "/\\<>.:%*?|\" ";
+  constexpr char *CharSet = "/\\<>.:%*?|\" ";
```

Comment:
You seemed to change the type of `CharSet` accidentally.
```suggestion
  constexpr const char *CharSet = "/\\<>.:%*?|\" ";
```

Reply:
Thanks for catching this.